PaddleCV/object_detection/ppdet/data/source/voc_loader.py

```diff
@@ -243,25 +243,25 @@ def load(anno_path, sample_num=-1, use_default_label=True):
 
 def pascalvoc_label():
     labels_map = {
-        'aeroplane': 1,
-        'bicycle': 2,
-        'bird': 3,
-        'boat': 4,
-        'bottle': 5,
-        'bus': 6,
-        'car': 7,
-        'cat': 8,
-        'chair': 9,
-        'cow': 10,
-        'diningtable': 11,
-        'dog': 12,
-        'horse': 13,
-        'motorbike': 14,
-        'person': 15,
-        'pottedplant': 16,
-        'sheep': 17,
-        'sofa': 18,
-        'train': 19,
-        'tvmonitor': 20
+        'car': 1,
+        'horse': 2,
+        'person': 3,
+        'bottle': 4,
+        'dog': 5,
+        'chair': 6,
+        'boat': 7,
+        'bicycle': 8,
+        'cat': 9,
+        'diningtable': 10,
+        'bird': 11,
+        'motorbike': 12,
+        'sofa': 13,
+        'train': 14,
+        'cow': 15,
+        'bus': 16,
+        'pottedplant': 17,
+        'tvmonitor': 18,
+        'aeroplane': 19,
+        'sheep': 20
```

Comment:
dot not change this order. 

see https://github.com/NVIDIA/DIGITS/blob/master/examples/semantic-segmentation/pascal-voc-classes.txt

https://github.com/PaddlePaddle/models/blob/develop/PaddleCV/ssd/data/pascalvoc/label_list

Reply:
Done